Doc/whatsnew/3.14.rst

```diff
@@ -966,6 +967,18 @@ mimetypes
     Add OpenDocument ``.odg``, ``.odp``, ``.ods`` and ``.odt`` types
   * `W3C <https://www.w3.org/TR/epub-33/#app-media-type>`__:
     Add EPUB ``application/epub+zip`` (``.epub``)
+  * Add 7z ``application/x-7z-compressed`` (``.7z``)
```

Comment:
Keep this list sorted? It's unclear if the new additions fall under the 'de facto' category or if some of them have standards bodies, like above.

Reply:
Updated!

It may be clearer to rewrite a bunch of these addition bullets together, even though they were contributed across a few issues. But that can be future copyediting.